docs/dap/user-defined-dap/python-debugpy.md

```diff
@@ -105,9 +105,15 @@ in the IntelliJ console. Read [Python debugging in VS Code](https://code.visuals
 ## Set Breakpoint
 
 After applying the run configuration, you should set a breakpoint to files which matches file mappings.
-Set a breakpoint in the `test.py` file:
+Setting a breakpoint requires two steps due to the default Python Line Breakpoint type provided by PyCharm:

```

Comment:
I am sorry to be not clear. When you are not using PyCharm like IJ community you have not this issue.

Please separate your content in 2 sections : with and without PyCharm.

I say if you are using PyCharm you need 2 steps otherwise you can just set the breakpoint in one step with community